api/src/main/resources/org/openmrs/liquibase/updates/liquibase-update-to-latest-2.5.x.xml

```diff
@@ -268,4 +268,97 @@
+
+	<changeSet id="2021-24-10-1145-TRUNK-6038" author="miirochristopher">
+		<preConditions onFail="MARK_RAN" onFailMessage="Table diagnosis_attribute already exists">
+			<not>
+				<tableExists tableName="diagnosis_attribute" />
+			</not>
+		</preConditions>
+		<comment>Creating diagnosis_attribute table</comment>
+		<createTable tableName="diagnosis_attribute">
+			<column name="diagnosis_attribute_id" type="int" autoIncrement="true">
+				<constraints primaryKey="true" nullable="false" />
+			</column>
+			<column name="diagnosis_id" type="int"><constraints nullable="false" /></column>
+			<column name="attribute_type_id" type="int"><constraints nullable="false" /></column>
+			<column name="value_reference" type="text"><constraints nullable="false" /></column>
+			<column name="uuid" type="char(38)"><constraints nullable="false" unique="true" /></column>
+			<column name="creator" type="int"><constraints nullable="false"/></column>
+			<column name="date_created" type="datetime"><constraints nullable="false"/></column>
+			<column name="changed_by" type="int"/>
+			<column name="date_changed" type="datetime" />
+			<column name="voided" type="boolean" defaultValueBoolean="false">
+				<constraints nullable="false"/>
+			</column>
+			<column name="voided_by" type="int" />
+			<column name="date_voided" type="datetime" />
+			<column name="void_reason" type="varchar(255)" />
+		</createTable>
+		<addForeignKeyConstraint constraintName="diagnosis_attribute_diagnosis_fk" baseTableName="diagnosis_attribute" baseColumnNames="diagnosis_id" referencedTableName="encounter_diagnosis" referencedColumnNames="diagnosis_id" />
+		<addForeignKeyConstraint constraintName="diagnosis_attribute_attribute_type_id_fk" baseTableName="diagnosis_attribute" baseColumnNames="attribute_type_id" referencedTableName="diagnosis_attribute_type" referencedColumnNames="diagnosis_attribute_type_id" />
+		<addForeignKeyConstraint constraintName="diagnosis_attribute_creator_fk" baseTableName="diagnosis_attribute" baseColumnNames="creator" referencedTableName="users" referencedColumnNames="user_id" />
+		<addForeignKeyConstraint constraintName="diagnosis_attribute_changed_by_fk" baseTableName="diagnosis_attribute" baseColumnNames="changed_by" referencedTableName="users" referencedColumnNames="user_id" />
+		<addForeignKeyConstraint constraintName="diagnosis_attribute_voided_by_fk" baseTableName="diagnosis_attribute" baseColumnNames="voided_by" referencedTableName="users" referencedColumnNames="user_id" />
+	</changeSet>
+
+	<changeSet id="2021-01-11-0700-TRUNK-6038" author="miirochristopher">
+		<preConditions onFail="MARK_RAN" onFailMessage="Table encounter_diagnosis or column rank does not exist.">
+			<tableExists tableName="encounter_diagnosis" />
+			<and>
+				<columnExists tableName="encounter_diagnosis" columnName="rank" />
+			</and>
+		</preConditions>
+		<comment>Renaming column rank to dx_rank because rank is a reserved word in MySQL 8.0.2 and later</comment>
```

Comment:
Since rank already existed even before this ticket, i would use a separate ticket and pull request for its rename.

Reply:
@dkayiwa, understood but kindly help me through creating the tickect.

Reply:
@miirochristopher feel free to ask on talk.

Reply:
@dkayiwa would this suffice? kindly let me know if I need to change anything.